libr/util/pj.c

```diff
@@ -90,32 +90,42 @@ R_API PJ *pj_k(PJ *j, const char *k) {
 }
 
 R_API PJ *pj_kn(PJ *j, const char *k, ut64 n) {
-	pj_k (j, k);
-	pj_n (j, n);
+	if (n) {
+		pj_k (j, k);
+		pj_n (j, n);
+	}
 	return j;
 }
 
 R_API PJ *pj_kd(PJ *j, const char *k, double d) {
-	pj_k (j, k);
-	pj_d (j, d);
+	if (d) {
+		pj_k (j, k);
+		pj_d (j, d);
+	}
 	return j;
 }
 
 R_API PJ *pj_ki(PJ *j, const char *k, int i) {
-	pj_k (j, k);
-	pj_i (j, i);
+	if (i && i != UT64_MAX) {
```

Comment:
`i` cannot be `UT64_MAX` because it's likely a signed 32bit meanwhile `UT64_MAX` is unsigned 64bit value.